x-pack/plugins/lens/public/xy_visualization/xy_config_panel/shared/line_style_settings.tsx

```diff
@@ -40,7 +40,7 @@ export const LineStyleSettings = ({
           defaultMessage: 'Line',
         })}
       >
-        <EuiFlexGroup gutterSize="s" justifyContent="spaceBetween">
+        <EuiFlexGroup gutterSize="s" justifyContent="spaceBetween" responsive={false}>
```

Comment:
The line style `EuiButtonGroup` that is a descendant of this flex group should have its `isFullWidth` prop removed. Also, the `EuiButtonGroup`'s parent `EuiFlexItem` should have it's `grow` prop set to `{false}`. This will ensure the button group never grows bigger than the size of its contents (i.e. the icons).